src/components/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useContext } from 'react';
+import { Todo } from '../types/Todo';
+import { TodosContext } from '../contexts/TodosContext';
+
+export const Header: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const { todos, setTodos } = useContext(TodosContext);
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    if (title.trim()) {
+      const newTodo: Todo = {
+        id: +(new Date()),
+        title,
+        completed: false,
+      };
+
+      setTitle('');
+
+      setTodos([...todos, newTodo]);
+    }
+  };
+
+  return (
+    <header className="header">
+      <h1>todos</h1>
+
+      <form onSubmit={handleSubmit}>
+        <input
```

Comment:
Also submit form on input blur event

Reply:
Thanks for the suggestion, but I think this is an unnecessary or even redundant feature, the readme only says to add a task when submitting the form (Enter).

Reply:
That's fine, its just the way the [todoapp](https://todomvc.com/examples/vanillajs/) given as example handles the case

Reply:
Now I see, thanks again for the advice, I was careless!